smithy-swift-codegen/src/test/kotlin/HttpProtocolClientGeneratorTests.kt

```diff
@@ -37,25 +37,14 @@ class HttpProtocolClientGeneratorTests {
                 }
             
                 public convenience init() throws {
-                    let config = try RestJsonProtocol.RestJsonProtocolConfiguration()
+                    let config = try RestJsonProtocol.RestJsonProtocolConfiguration("Rest Json Protocol", "RestJsonProtocolClient")
```

Comment:
Clarifying question: is there some background for the additional parameters here?

Reply:
I have updated the generic client config to be `DefaultSDKRuntimeConfiguration` class in ClientRuntime. The class constructor takes service name and client name.